test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java

```diff
@@ -445,6 +445,21 @@ public void highlightTest () throws Exception{
 
         //highlight in objects
         assertThat("incorrect highlight", actual.getHighlight(), is(expected.getHighlight()));
+
```

Comment:
This should be a unit test, not an integration test. In the interests of getting this out, do you solemnly swear to switch it to an unit test when the API changes are done?

Reply:
yes :)